assets/js/components/declaratie/Declaratie.vue

```diff
@@ -4,73 +4,91 @@
     <div class="voortgang">
       <div
         class="fase concept"
-        :class="{'active': declaratie.status === 'concept', 'done': declaratie.status !== 'concept'}"
+        :class="{
```

Comment:
Heb niet perse iets tegen het opnieuw autoformatten van deze bestanden, maar deze PR is al van redelijk formaat. Dus misschien is het beter om deze autoformats los te doen.

Reply:
Welke formatter heb je gebruikt? Ik krijg mijn PhpStorm niet zo ver om het op deze manier te formatten.

Reply:
Ik gebruik Prettier voor het formatten, maar ik zal het voor nu uitzetten. Ik zou het wel chill vinden als dat later een keer helemaal word gedaan want dat maakt de code toch meer leesbaar

Reply:
Je mag het ook zo instellen dat we het allemaal gebruiken hoor 😄. Maar als er verschillende standaarden zijn wordt het een beetje een bende.

Reply:
Volgens mij kan je phpstorm instellen om automatisch prettier te gebruiken, lijkt me mega nice als dat werkt